templates/user.html

```diff
@@ -15,21 +15,25 @@
   <meta property="og:description" content="See all your Open Source contributions in one place."/>
   <!-- Open Graph End -->
 
+  {{if (ne .ResponseType "iframe")}}
   <!-- Twitter Card Start -->
```

Comment:
@lex111 I thought it would be a good idea but then I don't want to maintain two different pr item templates.

@Overtorment can you see if you can move the pr list to a separate template and then import it? If that is possible then keeping two separate templates would be more maintainable. What do you think? You want to give it a shot?